functions.php

```diff
@@ -34,34 +38,71 @@ public function __construct() {
 		add_action( 'init', array( $this, 'register_taxonomies' ) );
 		add_action( 'cmb2_admin_init', array( $this, 'register_header_metabox' ) );
 		add_action( 'pre_get_posts', array( $this, 'tags_support_query' ) );
+		add_action( 'admin_init', array( $this, 'add_copyright_text' ) );
 
 		// Default actions are in: https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-includes/default-filters.php#L0
 		remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
 		remove_action( 'wp_head', 'wp_generator' );
 		remove_action( 'wp_print_styles', 'print_emoji_styles' );
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_parent_styles') );
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_parent_styles' ) );
 		parent::__construct();
 	}
 
+	/**
+	 * Show copyright text field.
+	 *
+	 * @param array $args
+	 */
+	public function cp_show_settings( $args ) {
```

Comment:
I'd use verbose name for function here too in line with other function names. So ex `copyright_show_settings` instead of `cp_show_settings`.

Reply:
Done